ui/src/app/settings/components/settings-overview/settings-overview.tsx

```diff
@@ -13,8 +13,8 @@ const settings = [
         path: './repos'
     },
     {
-        title: 'Certificates',
-        description: 'Configure certificates for connecting Git repositories',
+        title: 'Certificates and known hosts',
```

Comment:
Can this also be 'Repository certificate and known hosts'. Just to make it consistent with the other settings.

Reply:
@saumeya made the change. 
please note - I changed both title and description to Repository certificates